recipes-ni/ni-systemimage/files/nisystemimage

```diff
@@ -366,6 +384,15 @@ init_fs_get () {
 	fi
 }
 
+init_fs_getall () {
```

Comment:
Is this function, or `init_fd_getall()` called anywhere?

Reply:
Yes, `init_fd` calls `init_fd_$MODE` which in our case would be `init_fd_getall`.